docs/building-apps/abi-compatibility.rst

```diff
@@ -0,0 +1,22 @@
+.. _label-binary-compatibility:
+
+ABI compatibility to previous versions of Open MPI
+==================================================
+
+Open MPI v5.0 intends to maintain Application Binary Interface (ABI)
+compatibility to the last major Open MPI release. Technically, an
+application compiled with Open MPI 4.1 can be executed with Open MPI
+5.0 without having to recompile the application.
```

Comment:
The ABI guarantees go all the way back to v4.0.0 -- not just v4.1.x.  I.e., the "last major release" is 4, not 4.1.

```suggestion
The Open MPI |ompi_series| series maintains Application Binary Interface (ABI)
compatibility for the C MPI bindings to the last major Open MPI release. Specifically, an
application compiled with Open MPI v4.x can be executed with Open MPI
|ompi_series| without having to recompile the application.

.. important:: ABI is maintained for *most* of the Fortran MPI bindings, too -- see below for additional information.
```